cla_public/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -1512,11 +1544,11 @@ msgid "These cookies need to be on to give you the best experience using the ser
 msgstr "Mae angen i’r cwcis hyn fod ar waith er mwyn i chi gael y profiad gorau wrth ddefnyddio’r gwasanaeth."
 
 #: cla_public/templates/_cookie-consent-form.html:37
-msgid "Read more about cookies on 'Check if you can get legal aid'"
+msgid "Read more about cookies on ‘Check if you can get legal aid’"
 msgstr "Darllenwch fwy am ddefnyddio cwcis ar ‘Gwirio os ydych yn gymwys i gael cymorth cyfreithiol’"
 
 #: cla_public/templates/_cookie-consent-form.html:42
-msgid "Cookies used to improve 'Check you can get legal aid'"
+msgid "Cookies used to improve ‘Check you can get legal aid’"
 msgstr "Cwcis sy’n cael eu defnyddio i wella ‘Gwirio os ydych yn gymwys i gael cymorth cyfreithiol’"
```

Comment:
- For lines 1551 to 1552, will you not need to also change the Welsh translation for this? 
- Will you not need to also change other references e.g. like on line 1547 and 1548, so that it will read >Check you can get legal aid and not > Check if you can get legal aid

Reply:
The Welsh was already correct in these instances, the smart quotes were only missing from the English.

Regarding `Check you can get legal aid` vs `Check if you can get legal aid`, I have corrected this after talking with Josh.